### PR TITLE
Updates use cases relating to appointments in DG 

### PR DESCRIPTION
**Pull request by woon17**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -410,40 +410,99 @@ Use case ends.
...
 [discrete]
-=== Use case: Set next appointment (UC4)
+=== Use case: Find patients appointment (UC5)
 
 *MSS*
 
-1.  Patient wants to set next appointment
-2.  User requests to set the patient's appointment date
-3.  ClerkPro changes the patient's appointment date
+1.  Actor requests to find patient's appointment list
```
Comment (SakuraBlossom) [inline docs/DeveloperGuide.adoc]: Please standardise to "user"

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -410,40 +410,99 @@ Use case ends.
...
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: Change appointment date (UC4)
+Pre-condition: Patient’s appointment exists
+
+*MSS*
+
+1.  Patient wants to Change a current appointment date
```
Comment (SakuraBlossom) [inline docs/DeveloperGuide.adoc]: I believe your MSS should begin at the user's interaction with the system.

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (woon17) [reply docs/DeveloperGuide.adoc]: okay

File: docs/DeveloperGuide.adoc (hunk 2 above)
Comment (woon17) [reply docs/DeveloperGuide.adoc]: okay

**Pull request merged**